src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String color;
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Circle has " + color + " and radius " + radius
+                + ". Area is " + getArea());
+    }
+
+    @Override
+    public int getArea() {
+        return (int) (Math.PI * Math.pow(radius, 2));
```

Comment:
When you cast result of this expression to int value can be lost, so maybe it will be better to make areaCalculator return double